### DISCUSSION TITLE
怎么删除索引？尝试建立了索引，服务器占用太大

### DISCUSSION THREAD

**Discussion by H-huihui** [AlistGo/alist#2614] [General]
索引的对象计数:1303795 
服务器直接100%了🤦

求问怎么删除掉这部分索引？

**Comment by nxjonson** [comment]
索引前备份了data.db:1，备份文件覆盖data目录里的data.db文件；2，alist管理中“搜索索引”选择无，保存；3，重启alist

**Reply by H-huihui** [reply]
感谢提供思路，我用的别的逻辑处理的。重启了alist，在alist管理中“搜索索引”选择无，保存，就可以了

**Reply by makergit365** [reply]
这里面两种索引啥区别啊，构建为啥好慢

**Reply by nxjonson** [reply]
https://alist.nn.ci/zh/guide/advanced/search.html#%E5%A6%82%E4%BD%95%E4%BD%BF%E7%94%A8

**Comment by gitcook** [comment]
没备份，怎么删除呢，alist插件重启没用
